workspace/apps/pidp/src/app/features/shell/components/portal-dashboard/portal-dashboard.component.ts

```diff
@@ -170,6 +170,15 @@ export class PortalDashboardComponent implements IDashboard, OnInit {
         },
         'help_outline',
       ),
+      new DashboardRouteMenuItem(
```

Comment:
The new `nav-menu` component will be used moving forward so there is no need to update `portal-dashboard.component.ts`. We probably will be removing this component.